vita3k/gui/src/main_menubar.cpp

```diff
@@ -93,27 +95,28 @@ static void draw_emulation_menu(GuiState &gui, EmuEnvState &emuenv) {
     }
 }
 
-static void draw_debug_menu(DebugMenuState &state) {
-    if (ImGui::BeginMenu("Debug")) {
-        ImGui::MenuItem("Threads", nullptr, &state.threads_dialog);
-        ImGui::MenuItem("Semaphores", nullptr, &state.semaphores_dialog);
-        ImGui::MenuItem("Mutexes", nullptr, &state.mutexes_dialog);
-        ImGui::MenuItem("Lightweight Mutexes", nullptr, &state.lwmutexes_dialog);
-        ImGui::MenuItem("Condition Variables", nullptr, &state.condvars_dialog);
-        ImGui::MenuItem("Lightweight Condition Variables", nullptr, &state.lwcondvars_dialog);
-        ImGui::MenuItem("Event Flags", nullptr, &state.eventflags_dialog);
-        ImGui::MenuItem("Memory Allocations", nullptr, &state.allocations_dialog);
-        ImGui::MenuItem("Disassembly", nullptr, &state.disassembly_dialog);
+static void draw_debug_menu(GuiState &gui, DebugMenuState &state) {
+    auto &lang = gui.lang.main_menubar.debug;
+    if (ImGui::BeginMenu(lang["title"].c_str())) {
+        ImGui::MenuItem(lang["threads"].c_str(), nullptr, &state.threads_dialog);
+        ImGui::MenuItem(lang["semaphores"].c_str(), nullptr, &state.semaphores_dialog);
+        ImGui::MenuItem(lang["mutexes"].c_str(), nullptr, &state.mutexes_dialog);
+        ImGui::MenuItem(lang["lightweight_mutexes"].c_str(), nullptr, &state.lwmutexes_dialog);
+        ImGui::MenuItem(lang["condition_variables"].c_str(), nullptr, &state.condvars_dialog);
+        ImGui::MenuItem(lang["lightweight_condition_variables"].c_str(), nullptr, &state.lwcondvars_dialog);
+        ImGui::MenuItem(lang["event_flags"].c_str(), nullptr, &state.eventflags_dialog);
+        ImGui::MenuItem(lang["memory_allocations"].c_str(), nullptr, &state.allocations_dialog);
+        ImGui::MenuItem(lang["disassembly"].c_str(), nullptr, &state.disassembly_dialog);
```

Comment:
for this, i no know 🤔 , what do you thinks of this @Macdu ?
For me, if i have never wanted translated it, is because is literally only for dev, so no idea if is needed, but so i ask your opinion

Reply:
I do agree that translating this feels very useless from my point of view (I think at most 4 people use this menu, myself included).
But if someone else went through the trouble of adding language support for these menus, I don't really see a reason not to accept it.